feature-configs/5g-ref-nw/clusters/otwaon1234rd/cu-up/role-hardware/performance/kustomization.yaml

```diff
@@ -0,0 +1,14 @@
+# Base profile for du-large flavor
```

Comment:
Cleanup the comment.  This is for the CU.

Reply:
fixed